SECURITY.md

```diff
@@ -0,0 +1,111 @@
+# Security Policy
+
+## Table of contents
+
+1. Introduction
+2. Scope
+3. Supported Versions
+4. Reporting a Vulnerability
+5. Guidelines
+6. Test methods
+7. Authorization
+8. Questions
+
+## Introduction
+
+Codenotary, September 16th 2022
+
+*This security policy is based on [this template](https://www.cisa.gov/vulnerability-disclosure-policy-template).*
+
+Codenotary is committed to ensuring the security of the global public by protecting their information. This policy is intended to give security researchers clear guidelines for conducting vulnerability discovery activities and to convey our preferences in how to submit discovered vulnerabilities to us.
+
+This policy is addressed to people using the immudb and / or contributing to the project. It describes **what systems and types of research** are covered under this policy, **how to send** us vulnerability reports, and **how long** we ask security researchers to wait before publicly disclosing vulnerabilities.
+
+We encourage you to contact us to report potential vulnerabilities in our systems.
+
+## Scope
+
+This policy applies to the following systems and services:
+
+- immudb database core application, 
+- immuadmin and immuclient applications
+- golang immudb SDK
+
+Source code of the above tools [is available on GH](https://github.com/codenotary/immudb).
+Also:
+
+- Java immudb SDK ([repo link](https://github.com/codenotary/immudb4j))
+- Python immudb SDK ([repo link](https://github.com/codenotary/immudb-py))
+- .Net immudb SDK ([repo link](https://github.com/codenotary/immudb4net))
+
+Any service not expressly listed above, such as any connected services, are excluded from scope and are not authorized for testing. Additionally, vulnerabilities found in systems from our vendors fall outside of this policy’s scope and should be reported directly to the vendor according to their disclosure policy (if any). If you aren’t sure whether a system is in scope or not, contact us at (immudb-security at codenotary.com) before starting your research (or at the security contact for the system’s domain name listed in the .gov WHOIS).
+
+Though we develop and maintain other internet-accessible systems or services, we ask that active research and testing only be conducted on the systems and services covered by the scope of this document. If there is a particular system not in scope that you think merits testing, please contact us to discuss it first. We will increase the scope of this policy over time.
+
+Other Immudb SDK repositories that are not covered by this policy:
+
+- Ruby on rails SDK ([repo link](https://github.com/ankane/immudb-ruby))
+- Nodejs SDK ([repo link](https://github.com/user3232/node-immu-db))

```

Comment:
This is now merged into https://github.com/codenotary/immudb-node - shouldn't it be included in our policy then too ?